File: TourMate.FeedbackService.Api/TourMate.FeedbackService.Repositories/Models/Feedback.cs
using System;
using System.Collections.Generic;

namespace TourMate.FeedbackService.Repositories.Models;

public partial class Feedback
{
    public int FeedbackId { get; set; }

    public int CustomerId { get; set; }

    public int TourGuideId { get; set; }

    public DateTime CreatedDate { get; set; }

    public string Content { get; set; } = null!;

    public int Rating { get; set; }

    public bool IsDeleted { get; set; }

    public DateTime UpdatedAt { get; set; }

    public int InvoiceId { get; set; }
}

File: model/dto/request/feedback.go
package request

type GetFeedbacksRequest struct {
	Request     SearchPaginationRequest `json:"request"`
	CustomerId  int                     `json:"customer_id" form:"customer_id" validate:"min=1"`
	TourGuideId int                     `json:"tour_guide_id" form:"tour_guide_id" validate:"min=1"`
	InvoiceId   int                     `json:"invoice_id" form:"invoice_id" validate:"min=1"`
	Rating      int                     `json:"rating" form:"rating" validate:"min=1"`
	IsDeleted   *bool                   `json:"is_deleted" form:"is_deleted"`
}
